website/docs/Installation/macos-install.md

```diff
@@ -5,8 +5,6 @@ sidebar_position: 3
 # MacOS
 ## Installing Podman Desktop on MacOS using .dmg file
```

Comment:
I think this section needs to be re-wrote. 
Should be clear and guiding. Maybe move to a bullet list with clear instruction.
Also explain what happens when starting podman desktop for the first time. There are the pre-flight checks + what do we do when initializing.
Also important - once installed and setup, there should be CAT on what to do next!

Reply:
CAT?